server/backend/testspg/authorization/test_jwt_validation.py

```diff
@@ -20,33 +17,27 @@
                                assert_error_response,
                                headers)
 from Crypto.PublicKey import RSA
-from mock_alchemy.mocking import UnifiedAlchemyMagicMock
+from testspg.constants import FAM_APPLICATION_ID
+
 
 LOGGER = logging.getLogger(__name__)
-endPoint = f"{apiPrefix}/fam_applications"
+endPoint = f"{apiPrefix}/fam_applications/{FAM_APPLICATION_ID}/user_role_assignment"
 
 
-def test_get_application_success(
-    test_client_fixture_unit: starlette.testclient.TestClient,
+def test_get_application_user_role_assignment_success(
+    test_client_fixture: starlette.testclient.TestClient,
```

Comment:
we don't have the fam_applications endpoint anymore, so use the get the user role assignment endpoint for testing instead. And we need to use the real db to check user's permission, cannot use the mocked db anymore.

Reply:
The test on "jwt_validation" previously uses "/fam_application" as endpoint to test on.
I think it does not matter the endpoints, the purpose was just to use any simple endpoint to check on jwt token parsing does work from the "jwt_validation.py".
Changing to "{FAM_APPLICATION_ID}/user_role_assignment" I think still can use the "_unit" test client, just need to provide different mocks; could be difficult in this case, with also the Depends() mock. 

However, I am wondering why switching using the "test_client_fixture", why this assert works "assert data == []"? I thought it should contains something?

Reply:
yeah, because currently we're using the get user role assignment endpoint, and it will check the permission of the requester, however if we mock the database, it won't find our requester from the database. But you're right, why the data is []... let me check...

Reply:
oh Ian, because all admin records are in the application admin table, for FAM it doesn't have any regular user access

Reply:
I see, thanks for clarification.